CORPSE_deriv.py

```diff
@@ -230,6 +249,14 @@ def CORPSE_deriv(SOM,T,theta,Ndemand,Ctransfer,params,claymod=1.0):
     derivs['AMC'] = atleast_1d(dmicrobeC['AM'])
     derivs['AMN'] = atleast_1d(dmicrobeN['AM'])
 
+    derivs['Int_ECMC'] = atleast_1d(-SOM['Int_ECMC'])
+    derivs['Int_AMC'] = atleast_1d(-SOM['Int_AMC'])
+    derivs['Int_N'] = atleast_1d(Ntransfer-Nlitter)
+
+    derivs['NfromNecro'] = atleast_1d(decomp['NecroN']*params['nup']['Necro'])
```

Comment:
You should check how you are plotting this result. What's being saved here is the rate of change of this quantity (g N/year), and the numbers in the simulation results will be integrated (cumulative) N derived from necromass (total g N uptake over the simulation). So if you want to calculate the fraction of N uptake derived from necromass in a single time step (instead of cumulatively over the whole simulation up until that time step), you will have to take the derivative of the results array.

Reply:
Thanks for the head-up. I took the derivative when calculating the plotted result. So I guess this is not an issue as you pointed out later :)